todoapp/src/components/Todo.vue

```diff
@@ -1,40 +1,210 @@
 <!-- eslint-disable vue/multi-word-component-names -->
-
 <script setup lang="ts">
-import type { TextString } from '../types/text'
+import type { TodoTypes } from '../types/text'
+import Incomplete from '../assets/eclipseblack.svg'
+import Complete from '../assets/completedTask.png'
 
 const props = defineProps<{
-  todos: TextString[]
-}>()
-const emits = defineEmits<{
-  (e: 'removeTodo', index: number): void
+  todos: TodoTypes[]
+  isMobile: boolean
 }>()
 
-function removeTodo(index: number) {
-  emits('removeTodo', index)
+function handleImportance(priority: number) {
+  if (priority === 0) {
+    return 'low'
+  } else if (priority === 1) {
+    return 'medium'
+  } else if (priority === 2) {
+    return 'high'
+  } else {
+    return ''
+  }
 }
-</script>
 
+function statusState(todo: TodoTypes) {
+  if (todo.status === false) {
```

Comment:
please name functions after action so this would be something like toggleTodoState()

Also you can just simply write: 
`
todo.status = !todo.status
`
Which will basically do the same thing